examples/cross_account/README.md

```diff
@@ -1 +1,32 @@
-# Cross Account Example
\ No newline at end of file
+# Cross Account Example
+
+This folder shows an example of Terraform code that uses the [datadog-agentless-scanner module](https://github.com/Datadog/terraform-module-datadog-agentless-scanner) to deploy a Datadog agentless scanner in a main [AWS](https://aws.amazon.com/) account and cross account role in your other accounts.
+
+In this example the full scanner setup is deployed in a single account dedicated for the scanning purpose.
+All your other accounts can be scanned from that account by deploying a single IAM role.
+
+
+## Quick start
+
+To deploy a Datadog agentless scanner:
+
+1. Go to the `scanner_account` folder
+1. Run `terraform init`.
+1. Run `terraform apply`.
+1. Set your Datadog [API key](https://docs.datadoghq.com/account_management/api-app-keys/).
+1. You can leave the `cross_account_delegate_arn` variable empty for now.
```

Comment:
What does "for now" mean? It doesn't seem like you populate this variable later on in the instructions. Can we remove this whole line?

Reply:
So when running the command, the customer will be asked to fill the variable. They don't have the value just yet and need to continue the process until they can have it.

The command will ask it like so:

```
var.cross_account_delegate_arn
  Specifies the cross accounts role ARN that the Datadog agentless scanner can assume
  
  Enter a value: 
```

Reply:
Understood. Then I think it might be good to leave out the "for now".

```suggestion
1. You can leave the `cross_account_delegate_arn` variable empty.
```